build.gradle.kts

```diff
@@ -89,6 +97,9 @@ dependencies {
     implementation("org.apache.httpcomponents.client5:httpclient5:5.3.1")
 
     implementation("org.apache.logging.log4j:log4j-core:2.23.1")
+    implementation("org.apache.logging.log4j:log4j-api:2.23.1")
```

Comment:
Please be careful with introducing new dependencies, could you indicate if these are all needed?